src/main/java/org/jboss/forge/addon/commands/ConfigSourceCommand.java

```diff
@@ -0,0 +1,99 @@
+package org.jboss.forge.addon.commands;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.forge.addon.config.ForgeConfigSourceProvider;
+import org.jboss.forge.addon.configuration.Configuration;
+import org.jboss.forge.addon.ui.command.AbstractUICommand;
+import org.jboss.forge.addon.ui.context.UIBuilder;
+import org.jboss.forge.addon.ui.context.UIContext;
+import org.jboss.forge.addon.ui.context.UIExecutionContext;
+import org.jboss.forge.addon.ui.input.UIInput;
+import org.jboss.forge.addon.ui.input.UISelectOne;
+import org.jboss.forge.addon.ui.metadata.UICommandMetadata;
+import org.jboss.forge.addon.ui.metadata.WithAttributes;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.result.Results;
+import org.jboss.forge.addon.ui.util.Categories;
+import org.jboss.forge.addon.ui.util.Metadata;
+import org.jboss.forge.furnace.util.Strings;
+
+
+public class ConfigSourceCommand extends AbstractUICommand {
```

Comment:
I think this could be named `ConfigSourceURLCommand` since it just adds URLs to the config